pyproject.toml

```diff
@@ -5,9 +5,12 @@ description = ""
 authors = ["Josip Mrden <josip.mrden@memgraph.io>"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.8.1"
```

Comment:
I think it would be good to have consistent packages versions across mage, gqla and here, I know they don't interfere but issues can easily occur